### PR TITLE
[v7r0] Fix for getting a DIRAC SE total space

### PR DESCRIPTION
**Pull request by fstagni**
BEGINRELEASENOTES

*DMS
FIX: StorageElementHandler: removed ignoreMaxStorageSize flag from getTotalDiskSpace and getFreeDiskSpace
ENDRELEASENOTES

### CODE REVIEW

**Comment by chaen** [comment]
I disagree with the logic. If such a maximum space is allowed, that's the maximum you'll get, so this is what you want to see. Why do you want to reverse the logic ?

**Comment by fstagni** [comment]
For LogSE in DIRAC certification, without this fix the Free space is larger that the Total.

**Comment by chaen** [comment]
OK I see. Looking more into detail, it dates from the original commit eafa271a46 which introduced partially this feature, but the wrong way around ! So I'd say that your fix is correct, but the default should be `False` in both methods `getTotalDiskSpace` and `getFreeDiskSpace`

File: Resources/Storage/DIPStorage.py [hunk 1]
```diff
@@ -535,11 +535,11 @@ def getOccupancy(self, *parms, **kws):
 
     rpc = RPCClient(self.url, timeout=120)
 
-    free = rpc.getFreeDiskSpace()
+    free = rpc.getFreeDiskSpace(True)
```
Comment (chaen) [inline Resources/Storage/DIPStorage.py]: Let's put `False`, it makes so much more sense ..

File: DataManagementSystem/Service/StorageElementHandler.py [hunk 2]
```diff
@@ -43,7 +43,7 @@
 from DIRAC.Resources.Storage.StorageBase import StorageBase
 
 
-BASE_PATH = ""
+BASE_PATH = "/"
```
Comment (andresailer) [inline DataManagementSystem/Service/StorageElementHandler.py]: Is this change necessary?

https://github.com/DIRACGrid/DIRAC/blob/e0fc1aa48f785babb845f5a364e82b033eb3c60c/DataManagementSystem/Service/StorageElementHandler.py#L118-L130

With `BASE_PATH="/"` this would no longer fail, while with the empty string it will.

Comment (fstagni) [reply DataManagementSystem/Service/StorageElementHandler.py]: It was just for the test. 
What if 
```
 BASE_PATH = getServiceOption(serviceInfo, "BasePath", BASE_PATH) 
```
is replaced by 
```
 BASE_PATH = getServiceOption(serviceInfo, "BasePath") 
```

Comment (chaen) [reply DataManagementSystem/Service/StorageElementHandler.py]: This PR starts to touch really a lot of things :-D Do you want me to check what the actual problem of LogSE is ? :-D

Comment (andresailer) [reply DataManagementSystem/Service/StorageElementHandler.py]: Yes changing the getServiceCall restores previous behavior.
Maybe add a comment to the BASE_PATH that it is not a constant, but a configurable option. It is not nice having this "default" value ("/") not actually being used.

**Comment by fstagni** [comment]
Review, please

**Comment by atsareg** [comment]
Is my impression wrong that the problems stem from the fact the MaxStorageSize parameter in the CS is defined not properly, too large for example ?

**Comment by fstagni** [comment]
Simplified + removed the `ignoreMaxStorageSize` flag
